debugger/erlide_debugger_23/src/dbg_icmd.erl

```diff
@@ -50,6 +51,10 @@
 %% specifies if the process should break.
 %%--------------------------------------------------------------------
 
+%% Common Test adaptation
```

Comment:
As there are no common tests for erlide_kernel what I can see, these type of changes are needed to debug a running common test?